modules/performanceplatform/manifests/proxy_vhost.pp

```diff
@@ -53,6 +53,13 @@
 
   $graphite_servername = regsubst($servername, '\.', '_', 'G')
 
+  if ($serveraliases != undef) {
+    validate_array($serveraliases)
+    $vhost_names = $serveraliases
+  } else {
+    $vhost_names = [$servername]
```

Comment:
I think

```
$vhost_names = [$servername]
```
And
```
$vhost_names = undef
```
evaluate to the same thing, as `$servername` is always undef in this case, and the latter seems clearer